controllers/administration.js

```diff
@@ -502,6 +624,55 @@ router.all('/students', permissionsHelper.permissionsChecker(['ADMIN_VIEW', 'STU
     });
 });
 
+router.patch('/helpdesk/:id', permissionsHelper.permissionsChecker('HELPDESK_VIEW'), getUpdateHandler('helpdesk'));
+router.get('/helpdesk/:id', permissionsHelper.permissionsChecker('HELPDESK_VIEW'), getDetailHandler('helpdesk'));
+router.delete('/helpdesk/:id', permissionsHelper.permissionsChecker('HELPDESK_VIEW'), getDisableHandler('helpdesk'));
+router.post('/helpdesk/:id', permissionsHelper.permissionsChecker("HELPDESK_VIEW"), getSendHelper('helpdesk'));
+router.all('/helpdesk', permissionsHelper.permissionsChecker('HELPDESK_VIEW'), function (req, res, next) {
+
+    const itemsPerPage = 10;
+    const currentPage = parseInt(req.query.p) || 1;
+    let title = returnAdminPrefix(res.locals.currentUser.roles);
+
+    api(req).get('/helpdesk', {
+        qs: {
+            $limit: itemsPerPage,
+            $skip: itemsPerPage * (currentPage - 1),
+            $sort: 'order'
+        }
+    }).then(data => {
+        const head = [
+            'Titel',
+            'Ist-Zustand',
+            'Soll-Zustand',
+            'Kategorie',
+            'Status',
+            'Anmerkungen',
+            ''
+        ];
+
+        const body = data.data.map(item => {
+            return [
+                truncate(item.subject),
+                truncate(item.currentState),
+                truncate(item.targetState),
+                dictionary[item.category],
+                dictionary[item.state],
+                truncate(item.notes),
+                getTableActionsSend(item, '/administration/helpdesk/', item.state)
+            ];
+        });
+
+        const pagination = {
+            currentPage,
+            numPages: Math.ceil(data.total / itemsPerPage),
+            baseUrl: '/administration/helpdesk/?p={{page}}'
+        };
+
+        res.render('administration/helpdesk', {title: title + 'Lehrer', head, body, pagination});
```

Comment:
![image](https://user-images.githubusercontent.com/9433996/29808842-fe33e592-8c99-11e7-9637-e5d2767bfe76.png)

"Administration: Lehrer" wohl besser in "Helpdesk" ändern?
Rolle oben?